server/datastore/create/seeds/01.db.js

```diff
@@ -0,0 +1,3 @@
+exports.seed = function(knex) {
+	return knex.raw('CREATE DATABASE reactseeddb');
```

Comment:
append the next two seeds in batch: `kenx.raw(...).then( ... )`
- CREATE DATABASE
- CREATE USER
- GRANT